GDJS/Runtime/runtimegame.js

```diff
@@ -63,6 +63,12 @@ gdjs.RuntimeGame = function(data, options) {
   this._debuggerClient = gdjs.DebuggerClient
     ? new gdjs.DebuggerClient(this)
     : null;
+
+  /**
+   * Property passed by the IDE when the current export is a preview.
+   * @type {boolean}
+   */
+  this.isPreview = spec.isPreview | false;
```

Comment:
`|| false`

Also can you make this "private" (_isPreview) and add a method `isPreview()` - to avoid tempting anyone to do a `isPreview = true`.